[2024-05-12T07:07:41.479+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-12T07:07:41.505+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v03.get_name scheduled__2023-07-21T00:00:00+00:00 [queued]>
[2024-05-12T07:07:41.513+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v03.get_name scheduled__2023-07-21T00:00:00+00:00 [queued]>
[2024-05-12T07:07:41.514+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-12T07:07:41.555+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_name> on 2023-07-21 00:00:00+00:00
[2024-05-12T07:07:41.561+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v03', 'get_name', 'scheduled__2023-07-21T00:00:00+00:00', '--job-id', '283', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpkt5v1qbd']
[2024-05-12T07:07:41.564+0000] {standard_task_runner.py:91} INFO - Job 283: Subtask get_name
[2024-05-12T07:07:41.562+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=9179) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-12T07:07:41.566+0000] {standard_task_runner.py:63} INFO - Started process 9184 to run task
[2024-05-12T07:07:41.615+0000] {task_command.py:426} INFO - Running <TaskInstance: our_dag_with_python_operator_v03.get_name scheduled__2023-07-21T00:00:00+00:00 [running]> on host e768b7f5ee51
[2024-05-12T07:07:41.688+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TonyA' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v03' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2023-07-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-21T00:00:00+00:00'
[2024-05-12T07:07:41.695+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-12T07:07:41.697+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-12T07:07:41.699+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: greet() missing 1 required positional argument: 'age'
[2024-05-12T07:07:41.714+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=our_dag_with_python_operator_v03, task_id=get_name, run_id=scheduled__2023-07-21T00:00:00+00:00, execution_date=20230721T000000, start_date=20240512T070741, end_date=20240512T070741
[2024-05-12T07:07:41.794+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 283 for task get_name (greet() missing 1 required positional argument: 'age'; 9184)
[2024-05-12T07:07:41.829+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-12T07:07:41.853+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-12T07:07:41.860+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
